templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -280,6 +280,7 @@ contents:
       # Remove OVS bridge "br-ex". Use the default NIC for cluster network.
 
       if nmcli connection show ovs-port-phys0 &> /dev/null; then
+        iface=$(nmcli connection show ovs-port-phys0 | grep connection.interface-name| awk '{print $2}')
```

Comment:
can we use get-values?

`nmcli --get-values connection.interface-name connection show ovs-port-phys0`

Reply:
Fixed.